File: datasets/IR-Plag/intellij-original-java-rest-kotlin/case-05/plagiarized/L1/01/L1.kt
package IR

import java.util.*

object L1 {
    @JvmStatic
    fun main(args: Array<String>) {
        print("Enter an integer: ")

        val input = Scanner(System.`in`)

        val number = input.nextInt()

        reverse(number)
    }

    fun reverse(number: Int) {
        var number = number
        while (number != 0) {
            val remainder = number % 10
            print(remainder)
            number = number / 10
        }
        println()
    }
}

File: datasets/IR-Plag/convertai-original-java-rest-python/case-05/plagiarized/L1/01/L1.py
import sys

def reverse(number):
    while number != 0:
        remainder = number % 10
        print(remainder, end='')
        number = number // 10
    print()

def main():
    print("Enter an integer: ", end='')
    number = int(input())
    reverse(number)

if __name__ == "__main__":
    main()

